demos/sample_azure_iot_ca_recovery/sample_azure_iot_ca_recovery.c

```diff
@@ -270,10 +315,38 @@ static void prvHandlePropertiesMessage( AzureIoTHubClientPropertiesResponse_t *
  */
 static uint32_t prvSetupNetworkCredentials( NetworkCredentials_t * pxNetworkCredentials )
 {
+    LogInfo( ( "Reading trust bundle from NVS\r\n" ) );
+    memset( ucRootCABuffer, 0, sizeof( ucRootCABuffer ) );
+    AzureIoTResult_t xResult = AzureIoTCAStorage_ReadTrustBundle( ucRootCABuffer,
+                                                                  sizeof( ucRootCABuffer ),
+                                                                  &ulRootCABufferWrittenLength,
+                                                                  ucRootCATrustBundleVersion,
+                                                                  sizeof( ucRootCATrustBundleVersion ),
+                                                                  &ulRootCATrustBundleVersionLength );
+    configASSERT( xResult == eAzureIoTSuccess );
```

Comment:
Are there situations where this is expected to return things other than `eAzureIoTSuccess`?

Reply:
Same for the other asserts later in this file

Reply:
Good callout for this one. I'll add a log message here saying that you need to run the flash sample to save the certs to NVS.